README.md

```diff
@@ -50,12 +50,16 @@ stack test --pedantic
 
 ### Running HLint
 
-All code in this repository should be as idiomatic as possible, so it is
-highly recommended to run `hlint` on your sources before opening a
-*pull request*.
+All code in this repository should be as idiomatic as possible, so we
+enforce in [Travis-CI] that it returns `No hints` when processed by
```

Comment:
Created a PR removing the brackets to fix the problem.

I think that the README deserves a rewrite, but I'm without ideas for now.